truffle/contract.go

```diff
@@ -134,6 +137,43 @@ func GetTruffleContracts(buildDir string, networkIDs []string, objects ...*model
 			}
 
 			absPath := node.AbsolutePath
+			if absPath[0] == '@' {
+				// local path
+				path, err := os.Getwd()
+				if err != nil {
+					return nil, 0, errors.Wrap(err, "failed getting working dir")
+				}
+
+				absPath = path + "/node_modules/" + absPath
+				doesNotExist := checkIfFileDoesNotExist(absPath)
+				if doesNotExist {
+					//global path - npm
+					cmd := exec.Command("npm", "root", "-g")
+					var out bytes.Buffer
+					cmd.Stdout = &out
+					err = cmd.Run()
+					if err != nil {
+						return nil, 0, errors.Wrap(err, "failed running npm")
+					}
+
+					globalNodeModule := strings.TrimSuffix(out.String(), "\n")
+					absPath = globalNodeModule + "/" + node.AbsolutePath
```

Comment:
Same as [this](https://github.com/Tenderly/tenderly-cli/pull/21/files#r470670488)